string source_map
string target_map
---
bool exists
float64 target_x
float64 target_y

